### PR TITLE
[FIX] Log warnings

### PR DESCRIPTION
**Pull request by ajayjaviya**
Task : https://www.druidoo.io/web#id=2053&action=361&active_id=28&model=project.task&view_type=form&menu_id=237

### CODE REVIEW

File: purchase_package_qty/models/stock_move_line.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ class StockMoveLine(models.Model):
 #     qty_done = fields.Float("Done (uom)")
     qty_done_package = fields.Float(
         "Done (package)", help="""The number of packages you've received.""",
-        digits_compute=dp.get_precision('Product Unit of Measure'))
+        digits=dp.get_precision('Product Unit of Measure'))
```
Comment (ivantodorovich) [inline purchase_package_qty/models/stock_move_line.py]: In odoo source they don't use `dp.get_precision`.
I think we can just remove it (and also the import)

https://github.com/odoo/odoo/blob/a36a693733579e1e8bdaa3aa00fcc482c6299acf/addons/stock/models/stock_move_line.py#L32

https://github.com/odoo/odoo/blob/a36a693733579e1e8bdaa3aa00fcc482c6299acf/addons/stock/models/stock_move_line.py#L33

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
`WARNING openerp_test odoo.addons.base.models.ir_model: Two fields (category_print_id, print_category_id) of product.product() have the same label: Print Category.`

Are we sure this is not a migration mistake? It's weird we have two fields with the same intention.
product_print_category was migrated and probably severly changed. Maybe this is something we didn't see?

**Comment by ajayjaviya** [comment]
> `WARNING openerp_test odoo.addons.base.models.ir_model: Two fields (category_print_id, print_category_id) of product.product() have the same label: Print Category.`
> 
> Are we sure this is not a migration mistake? It's weird we have two fields with the same intention.
> product_print_category was migrated and probably severly changed. Maybe this is something we didn't see?

print_category_id field is in `coop_default_pricetag` module (
https://github.com/druidoo/FoodCoops/blob/12.0/coop_default_pricetag/models/product_template.py#L148) but I am not sure about it's use

**Comment by ivantodorovich** [comment]
> > `WARNING openerp_test odoo.addons.base.models.ir_model: Two fields (category_print_id, print_category_id) of product.product() have the same label: Print Category.`
> > Are we sure this is not a migration mistake? It's weird we have two fields with the same intention.
> > product_print_category was migrated and probably severly changed. Maybe this is something we didn't see?
> 
> print_category_id field is in `coop_default_pricetag` module (
> https://github.com/druidoo/FoodCoops/blob/12.0/coop_default_pricetag/models/product_template.py#L148) but I am not sure about it's use

Can you test and check it out?
I think it's probably a mistake

**Comment by ajayjaviya** [comment]
> > > `WARNING openerp_test odoo.addons.base.models.ir_model: Two fields (category_print_id, print_category_id) of product.product() have the same label: Print Category.`
> > > Are we sure this is not a migration mistake? It's weird we have two fields with the same intention.
> > > product_print_category was migrated and probably severly changed. Maybe this is something we didn't see?
> > 
> > 
> > print_category_id field is in `coop_default_pricetag` module (
> > https://github.com/druidoo/FoodCoops/blob/12.0/coop_default_pricetag/models/product_template.py#L148) but I am not sure about it's use
> 
> Can you test and check it out?
> I think it's probably a mistake

@ivantodorovich : when rename module from `product_to_print` to `product_print_category` field `category_print_id` is also renamed to `print_category_id`

do we need to keep that change or need to rename field same as before in version 9 ?

**Comment by ivantodorovich** [comment]
Hi @ajayjaviya !

> do we need to keep that change or need to rename field same as before in version 9 ?

I don't understand exactly what you mean.
But we should have this module work with product_print_category, and be consistent with it.

According to what you say it seems we need to keep only `print_category_id`, right?

It also means we need to add a migration script to copy values from `category_print_id` to `print_category_id`. (I'll do it in the datbase migration script, no need to do it here)

**Comment by ajayjaviya** [comment]
> Hi @ajayjaviya !
> 
> > do we need to keep that change or need to rename field same as before in version 9 ?
> 
> I don't understand exactly what you mean.
> But we should have this module work with product_print_category, and be consistent with it.
> 
> According to what you say it seems we need to keep only `print_category_id`, right?
> 
> It also means we need to add a migration script to copy values from `category_print_id` to `print_category_id`. (I'll do it in the datbase migration script, no need to do it here)

yes ,so do i  rename `category_print_id` to `print_category_id` ?

**Comment by ivantodorovich** [comment]
Yes pls.
and analize why we're redefining this field because it seems it's already defined in product_print_category, right?

**Comment by ajayjaviya** [comment]
> Yes pls.
> and analize why we're redefining this field because it seems it's already defined in product_print_category, right?

@ivantodorovich yes sure I will analize it.

**Pull request merged**